### ISSUE TITLE
Calibrate does not work with latest hapi

### ISSUE THREAD

**Issue by jardakotesovec** [johnbrett/calibrate#2]
When starting hapi server I end up with:

```
Error: Invalid dependencies options (must be a string or an array of strings) {
  "boom": "~2.6.x",
  "code": "^1.3.0",
  "isdefined": "~1.x.x",
  "0" [1]: -- missing --
}
```

And its because - `decorate.attributes = require('../package.json');`
which contains dependencies property that confuses hapi.

It works ok when I replace it with just
`decorate.attributes = {name:'calibrate'}`

**Comment by johnbrett** [comment]
Hi @jardakotesovec, sincere apologies I missed this. Going to blame Github for not having a better way of seeing what issues you have open on your repos. I will investigate this later and resolve it.

Thanks for letting me know about this!

**Comment by johnbrett** [comment]
@jardakotesovec this is fixed as of v2.0.2 thanks to https://github.com/johnbrett/calibrate/pull/4. Thanks for raising the issue as well, and again sorry for taking so long!
